docs/customer.rb

```diff
@@ -0,0 +1,18 @@
+class Customer
```

Comment:
By convention, the folder containing the code is 'lib' (created by rspec); in this case 'docs' exists in order to contain the 'review.md' file.